src/System.Dynamic.Runtime/tests/Dynamic.Simple/DelegateTest.cs

```diff
@@ -24,5 +24,38 @@ public static void DelegateTest_RunTest()
             int result = d(4, 6);
             Assert.Equal(24, result);
         }
+
+        private static int Foo(string s, dynamic d, object o)
+        {
+            return s?.Length + d?.ToString()?.Length + o?.ToString()?.Length;
+        }
```

Comment:
@svick yeah, while it's just do-something-here-don't-care-what test code, there's a limit on how silly even code like that should be. Changed